### PR TITLE
added postNL letter support

### PR DESCRIPTION
**Pull request by scheric**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15389

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

- [x]  Approved by myself

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by hmmbob** [comment]
typo in multiple locations: packege --> package

File: source/_components/sensor.postnl.markdown [hunk 1]
```diff
@@ -13,20 +13,34 @@ ha_release: 0.69
 ha_iot_class: "Cloud Polling"
 ---
 
-The `postnl` platform allows one to track deliveries by [PostNL](https://www.postnl.nl) (Dutch Postal Services). To use this sensor, you need a [PostNL Account](https://jouw.postnl.nl). It is possible to add multiple accounts to your Home Assistant configuration.
+The `postnl` platform allows one to track packages and letters by [PostNL](https://www.postnl.nl) (Dutch Postal Services). To use this sensor, you need a [PostNL Account](https://jouw.postnl.nl). It is possible to add multiple accounts to your Home Assistant configuration.
 
-The sensor value shows the number of packages to be delivered. Each of the packages is available as an attribute.
+The package sensor value shows the number of packages to be delivered. Each of the packages is available as an attribute.
+The letters sensor value shows the number of letters to be deliverd.
```
Comment (iMicknl) [inline source/_components/sensor.postnl.markdown]: deliverd -> delivered

File: source/_components/sensor.postnl.markdown [hunk 2]
```diff
@@ -13,20 +13,34 @@ ha_release: 0.69
...
+
+```yaml
+sensor:
+  - platform: postnl
+    username: POSTNL_USERNAME
+    password: POSTNL_PASSWORD
+    letter: true
+```
+<p class='note'>
+  If you want to receve data from the letter sensor you need to enable it in the PostNL-app for Android ore IOS.
```
Comment (iMicknl) [inline source/_components/sensor.postnl.markdown]: receve -> receive
ore -> or
IOS -> iOS

File: source/_components/sensor.postnl.markdown [hunk 3]
```diff
@@ -13,20 +13,34 @@ ha_release: 0.69
...
+  See [this](https://www.postnl.nl/campagnes/mijn-post-in-de-app/) for more detaild instructions.
```
Comment (iMicknl) [inline source/_components/sensor.postnl.markdown]: detaild -> detailed

**Review by iMicknl — CHANGES_REQUESTED** [review]
Typo's

File: source/_components/sensor.postnl.markdown [hunk 4]
```diff
@@ -42,6 +56,11 @@ password:
   description: Account password of jouw.postnl.nl
   required: true
   type: string
+letter:
+  description: Enable ore disable letters
```
Comment (iMicknl) [inline source/_components/sensor.postnl.markdown]: ore -> or

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Looks good to me! Thanks, @scheric!
Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed as well.
Feel free to re-open when ready.

**Pull request closed without merging**